README.fr.md

```diff
@@ -62,6 +61,7 @@ Installez les dépendances :
 * pam **
 * imagemagick (requis pour la capture d'image avec swaygrab)
 * ffmpeg (requis pour la capture vidéo avec swaygrab)
+* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (requis pour man pages)
```

Comment:
This sounds really weird. I'd say "requis pour les pages man".

Reply:
I tried my best :')